kebechet/managers/manager.py

```diff
@@ -170,6 +170,21 @@ def create_pr(self, title: str, body: str, source_branch: str, target_branch: st
             fork_username=self.project.namespace if self.project.is_fork else None,
         )
 
+    def _git_commit_push(
+        self, commit_msg: str, branch_name: str, files: list, force_push: bool = False
+    ) -> None:
+        """Perform git push after adding files and giving a commit message."""
+        cur_branch = self.repo.active_branch
+        self.repo.git.checkout("HEAD", b=branch_name)
+        try:
+            self.repo.index.add(files)
+            self.repo.git.commit(f"-m {commit_msg}", "-s")
```

Comment:
Everythnig after `-m` becomes the commit message, so this should be at least:
```suggestion
            self.repo.git.commit(f"-m{commit_msg}", "-s")
```

However, one more thing about this: if the `commit_msg` is multi-line, it should check that there is a blank line after the first one... It might be that all commits right now only contain a title, but should we check to make this future-proof?

Reply:
The space is needed after `-m` . right ?
https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--mltmsggt

Reply:
I tested it without the space and it works fine. With the space, the commit title starts with a space.

I guess it can be more clear if the change is this instead:
```suggestion
            self.repo.git.commit(f"--message={commit_msg}", "--signoff")
```

Reply:
Can do. I'll make the flags more explicit